modules/core/src/main/java/org/apache/ignite/internal/managers/indexing/IndexesRebuildTask.java

```diff
@@ -103,30 +109,42 @@
                 }
             }
 
-            if (nonNull(err))
+            if (err != null)
```

Comment:
why not **else**?

Reply:
Because an error can appear when calling the **GridQueryProcessor#markAsRebuildNeeded** above.